redirects.csv

```diff
@@ -162,4 +163,5 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /healing/divorcecare,/groups/onsite-healing-groups/divorce-care,301!
 /healing/postpartum-healing,/groups/onsite-healing-groups/postpartum,301!
 /groups/onsite-groups/*,/groups/onsite,301!
+/groups/onsite,/groups/site-based,301!
```

Comment:
Let's make sure the slug for this category in production matches `site-based` vs. `site-based-groups` etc.

Reply:
Yes I will be removing `-groups` at the end of the slug in the cms once we do prod deploy.